spec/features/admin/subscriptions_spec.rb

```diff
@@ -224,17 +224,41 @@ def open_subscription_products_panel
         select2_select "Australia", from: "bill_address_country_id"
         select2_select "Victoria", from: "bill_address_state_id"
 
+        # Making sure that once set with empty it actually changes, and add the phone back
+        fill_in "bill_address_phone", with: ''
+        expect(page).to have_input "bill_address_phone", with: ''
+        expect(page).to have_content 'can\'t be blank', count: 1
```

Comment:
We tend to prefer using double quotes as a general style thing, and in this case it would mean you don't need to escape the single quote:
```diff
-- 'can\'t be blank'
++ "can't be blank"
```